sample_flask_code.py

```diff
@@ -0,0 +1,63 @@
+
+from flask import Flask,request,jsonify
+import json 
+import os
+
+app=Flask(__name__)
+
+data_file='data.json'
+
+try:
+     with open(data_file, 'r') as file:
+        tasks = json.load(file)
+            
+except FileNotFoundError:
+    with open(data_file, 'w') as file:
+     tasks = []
+     json.dump(tasks, file)
+
+@app.route('/www.files.com/', methods=['GET'])
+def get_tasks():
+    try:
+        with open(data_file, 'r') as file:
+            tasks = json.load(file)
+        return jsonify(tasks)
+    except Exception as e:
+        return jsonify({"error": str(e)}), 500
+
+
+@app.route('/www.files.com/', methods=['POST'])
+def create_task():
+    try:
+        new_task = request.get_json()
+
+        with open(data_file, 'r') as file:
+            tasks = json.load(file)
+
+        new_task['id'] = len(tasks) + 1
+        tasks.append(new_task)
+
+        with open(data_file, 'w') as file:
+            json.dump(tasks, file, indent=2)
+
+        return jsonify(new_task), 201
+
+    except Exception as e:
+        return jsonify({"error": str(e)}), 500   
+
+@app.route('/www.files.com/<int:task_id>', methods=['GET'])
+def get_task(task_id):
+    try:
+        with open(data_file, 'r') as file:
+            tasks = json.load(file)
+            task = next((t for t in tasks if t['id'] == task_id), None)
```

Comment:
Implementation of both,
1. List all tasks
2. Get task by id 
Should be in a single route function